### PR TITLE
Add Audit Log and Log DB security context selector and split screen into tabs

### PR DESCRIPTION
**Pull request by jinapurapu**
Screenshot from 2022-06-30 11-07-57
Screenshot from 2022-06-30 11-07-41

closes https://github.com/miniohq/engineering/issues/873
https://github.com/miniohq/engineering/issues/872

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]
Testd. LGTM.

File: operatorapi/tenants.go [hunk 1]
```diff
@@ -1451,10 +1460,13 @@ func getTenantLogsResponse(session *models.Principal, params operator_api.GetTen
 		tenantLoggingConfiguration.LogDBCPURequest = requestedDBCPU
 		tenantLoggingConfiguration.LogDBMemRequest = requestedDBMem
 	}
+
+	if minTenant.Spec.Log.SecurityContext != nil {
+	}
```
Comment (reivaj05) [inline operatorapi/tenants.go:1465]: Something missing in the if block?

**Review by reivaj05 — APPROVED** [review]

**Comment by cniackz** [comment]
Tested couple of fields, it looks good to me:

**Review by cniackz — APPROVED** [review]
LGTM

**Pull request merged**